lib/freedom_patches/rails6.rb

```diff
@@ -26,14 +26,10 @@ def initialize(disposition:, filename:)
           @filename = filename
         end
 
-        TRADITIONAL_ESCAPED_CHAR = /[^ A-Za-z0-9!#$+.^_`|~-]/
```

Comment:
wait, do we need any of this file if it is all included in rails 6? Why not just nuke this file?

Reply:
I backported everything below line 13 of that file and that stuff can be safely removed. I'm quite sure the whole file can be deleted.

Reply:
Nice catch! Thank you, I will remove that file